lib/commands/notegetoption.js

```diff
@@ -0,0 +1,20 @@
+import { ContextMenuCommandBuilder } from '@discordjs/builders'
```

Comment:
can you rename this file to just `note.js`

Reply:
wouldn't that be a little confusing with the slash command being in `notes.js`?

Reply:
maybe, notgetoption is confusing too tho. maybe notemenu then or something